core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1897,6 +1893,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NETWORK_ASYNC_CACHE_MANAGER_THREADS_MAX =
+      new Builder(Name.WORKER_NETWORK_ASYNC_CACHE_MANAGER_THREADS_MAX)
+          .setDefaultValue(8)
+          .setDescription("The maximum number of threads used to cache blocks asynchronously in "
+              + "the gRPC data server.")
```

Comment:
I don't think we need to specify the framework now that we only have 1.

Reply:
Removed.